tests/address-utils/test_address_utils.py

```diff
@@ -77,16 +100,19 @@ def test_is_checksum_address(value, expected):
 @pytest.mark.parametrize(
     "value,expected",
     [
-        ('0x52908400098527886E0F7030069857D2E4169EE7', False),
-        ('0x8617E340B3D01FA5F11F306F4090FD50E238070D', False),
-        ('0xde709f2102306220921060314715629080e2fb77', False),
-        ('0x27b1fdb04752bbc536007a920d24acb045561c26', False),
         ('0x5aAeb6053F3E94C9b9A09f33669435E7Ef1BeAed', True),
+        ('0xd3CdA913deB6f67967B99D67aCDFa1712C293601', True),
         ('0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359', True),
         ('0xdbF03B407c01E7cD3CBea99509d93f8DDDC8C6FB', True),
         ('0xD1220A0cf47c7B9Be7A2E6BA89F429762e7b9aDb', True),
+        ('0x52908400098527886E0F7030069857D2E4169EE7', False),
+        ('0x8617E340B3D01FA5F11F306F4090FD50E238070D', False),
+        ('0xde709f2102306220921060314715629080e2fb77', False),
+        ('0x27b1fdb04752bbc536007a920d24acb045561c26', False),
         ('0XD1220A0CF47C7B9BE7A2E6BA89F429762E7B9ADB', False),
-        ('0xd1220a0cf47c7b9be7a2e6ba89f429762e7b9adb', False)
+        ('0xd1220a0cf47c7b9be7a2e6ba89f429762e7b9adb', False),
+        (b'0XD1220A0CF47C7B9BE7A2E6BA89F429762E7B9ADB', False),
+        (b'0xd1220a0cf47c7b9be7a2e6ba89f429762e7b9adb', False),
```

Comment:
Another nice case would be a checksum address in `bytes` that would otherwise pass if it were decoded to `str`, like `b'0x5aAeb6053F3E94C9b9A09f33669435E7Ef1BeAed'`.